_drafts/technical-workflow.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title:  "The technical workflow of Etsy France"
+date:   2015-02-23 13:55:00
+categories: workflow
+tags: [workflow, jira, github, pr, review]
+author: antoine
+---
+
+Here at Etsy France we tried several organisations.  
+For some years now we fixed our organisation in scrum. To manage our workflow we use some useful tools that are interconnected: [Jira][jira], [Github][github], [HipChat][hipchat] and [Travis CI][travis].  
```

Comment:
For some years now **we have set** *our choice on* scrum (pour éviter les répétitions)